tests/vmi_networking_test.go

```diff
@@ -729,13 +729,32 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 					return vmi, nil
 				}
 
+				configureIpv6 := func(vmi *v1.VirtualMachineInstance) error {
+					err := console.RunCommand(vmi, "sudo dhclient -6 eth0", 30*time.Second)
+					if err != nil {
+						return err
+					}
+					err = console.RunCommand(vmi, "sudo ip -6 route add fd10:0:2::/120 dev eth0", 30*time.Second)
```

Comment:
I think we can reduce the timeout to 5sec, this command is async and does not wait for something to happen on the kernel. Same for the next one.

Regarding the `sudo`, I think the Fedora login is ending up with a `root` prompt.

Reply:
done.